spyderlib/utils/syntaxhighlighters.py

```diff
@@ -180,7 +199,8 @@ def setup_formats(self, font=None):
         
     def _check_color_scheme(self, color_scheme):
         if is_text_string(color_scheme):
-            assert color_scheme in COLOR_SCHEME_NAMES
+            pass
+#            assert color_scheme in COLOR_SCHEME_NAMES
         else:

```

Comment:
This is a very weird check and needs to be improved.

The method should expect one and one type only, but it works if it is a dictionary of color styles or a name of a style, and that is just bad design.